### DISCUSSION TITLE
need multiple allowed extensions when pick files

### DISCUSSION THREAD

**Discussion by BrentHuang** [flet-dev/flet#3257] [Q&A]
**Please Describe The Problem To Be Solved**
need multiple allowed extensions:
![image](https://github.com/flet-dev/flet/assets/2862064/c05eac83-fe34-40f9-91c7-be39cec3d00d)

now only one allowed extensions:
![image](https://github.com/flet-dev/flet/assets/2862064/99e0d78d-367d-4bd5-ab48-cfc515316fb8)

**Comment by ndonkoHenri** [comment]
- https://flet.dev/docs/controls/filepicker#file_type 

- https://flet.dev/docs/controls/filepicker#allowed_extensions

**Comment by BrentHuang** [comment]
can you show me the code, please?
I read the documentation in detail and I know the two parameters you mentioned.
I need more than one allowed extensions in the list above(For example, in the first picture, there are four). of couse, every allowed extensions has more than one file extension.

**Comment by ndonkoHenri** [comment]
When calling `pick_files()`:
- set `file_type=FilePickerFileType.CUSTOM`, 
- then set `allowed_extensions=["pdf", "svg", "jpg"]` (list of the the extensions to be allowed)

**Comment by BrentHuang** [comment]
@ndonkoHenri You probably don’t understand what I’m expressing. What I want is the effect like the one in the picture, with four instead of just one.
![image](https://github.com/flet-dev/flet/assets/2862064/5d50be77-3701-4a97-95b2-a6be4fe70d6e)

**Reply by ndonkoHenri** [reply]
Ah I see.
I honestly don't know if it is currently possible.
